Samples/Tests/Core/PartialCrashReport.swift

```diff
@@ -24,10 +24,24 @@
 // THE SOFTWARE.
 //
 
-#import <Foundation/Foundation.h>
+import Foundation
+/*
+ if let crash = report["crash"] as? [String:Any],
+    let error = crash["error"] as? [String:Any],
+    let reason = error["reason"] as? String {
+     parsedReason = reason
+ }
+ */
 
-@interface CrashTriggers : NSObject
+struct PartialCrashReport: Decodable {
```

Comment:
Is it supposed to be extended?

Reply:
Yes, and I would extend it separately. This PR is to setup a reliable foundation for integration tests